src/client/datascience/jupyter/jupyterSession.ts

```diff
@@ -239,7 +242,7 @@ export class JupyterSession extends BaseJupyterSession {
     }
 
     private async createSession(options: {
-        token?: CancellationToken;
```

Comment:
Because token is no longer nullable in `kernel.start` this method & others will never into situations where token === null